README.md

```diff
@@ -13,15 +13,59 @@ go get -v -u github.com/arr-ai/arrai/cmd/arrai
 arrai -h
 ```
 
-### Evaluate an expression
+On Unix-like platforms, you can also symlink a handy shortcut:
 
 ```bash
-arrai eval '41 + 1'
+ln -s arrai $(dirname $(which arrai))/ai
 ```
 
+## Learn
+
+Follow the [Arr.ai tutorial](docs/tutorial/tutorial.md) for a step by step guide
+into the world of arr.ai programming.
+
 See the [Introduction to Arr.ai](docs/intro.md) to learn more about the arr.ai
 language.
 
+### Arr.ai Examples
+
+1. [Snippets](docs/example.md)
+2. [More complete examples](examples)
+
+## Use
```

Comment:
Following commands also need to be listed here:
-    run, r        evaluate an arrai file
-    json, jx      Convert json to arrai
-    sync, s       sync local files to a server
-    transform, x  transform a stream of input data with an expression

Reply:
Good suggestion! Added placeholders to keep this PR small.